app/controllers/members_controller.rb

```diff
@@ -1,14 +1,20 @@
 class MembersController < ApplicationController
-  before_action :authenticate_user!, only: [:make_admin, :remove_admin, :destroy]
+  before_action :authenticate_user!, except: [:index]
   before_action :set_group
-  before_action :set_group_user, except: [:index]
+  before_action :set_group_user, except: [:index, :create]
   before_action :set_is_admin
   before_action :check_admin_rights, except: [:index]
```

Comment:
Hmmm, only group admins can add new users, right? If that's the case, we need to run the `check_admin_rights` before allowing the addition of a new member.